### DISCUSSION TITLE
chroma subsampling

### DISCUSSION THREAD

**Discussion by tazztone** [Tichau/FileConverter#283] [Ideas]
i just noticed when i convert a video that is 4:2:2 chroma subsampling to "mp4 low quality" it will remain 4:2:2.
i don't think it's possible to change such advanced parameters for that to compress the file even more. maybe a future feature? cheers

**Comment by djcurley89** [comment]
I raised this issue years ago actually... mainly because Macs cannot play mp4 videos with 422 chroma subsampling. On a PC, there is no issue, but when sending mp4s to people on macs they can not view. It was never addressed but I saw there's a new fork by someone else so may raise the issue there and see if could be fixed. Should just automatically change from 4:2:2 (prores for instance) to 4:2:0 when converting to mp4.
